react-nd/src/components/auth/LoginForm.module.scss

```diff
@@ -0,0 +1,26 @@
+@use '../../styles/variables';
```

Comment:
Naudokite globalius vite kintamuosius:
https://stackoverflow.com/questions/70849338/how-to-use-sass-global-variables-from-other-components-in-vite-vue3